src/main/java/com/teamabnormals/abnormals_core/core/util/ACHooks.java

```diff
@@ -0,0 +1,35 @@
+package com.teamabnormals.abnormals_core.core.util;
+
+import com.teamabnormals.abnormals_core.client.screen.SlabfishHatScreen;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screen.CustomizeSkinScreen;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.widget.Widget;
+import net.minecraft.client.gui.widget.button.Button;
+import net.minecraft.util.text.Color;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
```

Comment:
It'd be good to add some documentation to the top of this class. Something simple like saying it's used for calling methods in JS coremods.